### DISCUSSION TITLE
Unclear how this is different than plain clojurescript and its interoperability with the js ecosystem

### DISCUSSION THREAD

**Discussion by ukalbhor** [squint-cljs/cherry#87] [Q&A] [answered]
Can we add some points in the Readme on use-cases that cherry is useful for ?

**Comment by borkdude** [comment, marked as answer]
This is explained here:

https://github.com/squint-cljs/cherry#project-goals

Also you can watch my talk on this:

https://www.youtube.com/watch?v=oCd74TQ-gf4&t=1635s

After reading those resources, do you have suggestions how to make it even clearer?

**Reply by ukalbhor** [reply]
Thanks much.

I think the primary reason I was confused was why we need another compiler in the first place since we already have CLJS.

However, what helped was going through the `squint` readme where it states that the aim is a smaller bundle as well as simpler and more performant code.

Afterwards, understanding that cherry takes it a step ahead by making squint outputs compatible with es6 `mjs` modules made sense.

I want to apologize - I think I created this post in haste without giving the documentation a thorough read.
